src/Bridge/Symfony/DependencyInjection/Compiler/ConfigureDependencyPass.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Alice package.
+ *
+ * (c) Nelmio <hello@nelm.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Nelmio\Alice\Bridge\Symfony\DependencyInjection\Compiler;
+
+use RuntimeException;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class ConfigureDependencyPass implements CompilerPassInterface
```

Comment:
Maybe `CheckBundleDependenciesPass` would be more appropriate?

Reply:
Yeah, it's better, I was not inspired when I choose this name :D